app/pb_kits/playbook/pb_stat_value/stat_value.rb

```diff
@@ -2,61 +2,17 @@
 
 module Playbook
   module PbStatValue
-    class StatValue < Playbook::PbKit::Base
-      PROPS = %i[configured_classname
-                 configured_data
-                 configured_id
-                 configured_unit
-                 configured_value].freeze
+    class StatValue
+      include Playbook::Props
 
-      def initialize(classname: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     unit: default_configuration,
-                     value: default_configuration)
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_unit = unit
-        self.configured_value = value
-      end
-
-      def value
-        default_value(configured_value, 0)
-      end
-
-      def display_value
-        pb_title = Playbook::PbTitle::Title.new(size: 1, text: value)
-        ApplicationController.renderer.render(partial: pb_title, as: :object)
-      end
-
-      def unit
-        default_value(configured_unit, nil)
-      end
-
-      def display_unit
-        unless unit.nil?
-          pb_unit = Playbook::PbTitle::Title.new(size: 3, text: unit)
-          ApplicationController.renderer.render(partial: pb_unit, as: :object)
-        end
-      end
-
-      def kit_class
-        "pb_stat_value_kit"
-      end
-
-      def to_partial_path
-        "pb_stat_value/stat_value"
-      end
+      partial "pb_stat_value/stat_value"
 
-    private
+      prop :unit
```

Comment:
Does this one need a `Playbook::Props::String` and `default: nil`?

Reply:
He could have specified these. But since they are the default, it's okay to leave them off.

Reply:
So for a string prop, it is the default and automatically assumed by the code. No need to specify that or a default as nil.